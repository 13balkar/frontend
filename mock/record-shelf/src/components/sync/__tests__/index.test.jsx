import React from 'react';
import Sync from '..';
import { fireEvent, render } from '@testing-library/react';

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('Sync card', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Sync/>);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should call handle click when click on button', () => {
    const { getByText } = render(<Sync/>);
    const button = getByText('sync');
    fireEvent.click(button);
  });
});
