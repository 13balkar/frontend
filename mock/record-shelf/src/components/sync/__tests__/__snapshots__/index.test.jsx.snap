// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sync card should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sync"
  >
    <p>
      :((
    </p>
    <p>
      seems a bit empty here
    </p>
    <button>
      sync
    </button>
  </div>
</DocumentFragment>
`;
